### ISSUE TITLE
Schema validation of multipleOf throws error for values less than one

### ISSUE THREAD

**Issue by yasammez** [OpenLiberty/open-liberty#11085]
**Describe the bug**
Validating a schema which includes a multipleOf property set greater than zero but smaller than 1 produces a log output 
```
[2/24/20 23:16:30:999 CET] 0000003c com.ibm.ws.microprofile.openapi.ApplicationProcessor         E CWWKO1650E: Validation of the OpenAPI document produced the following error(s): 
 - Message: The Schema Object must have the "multipleOf" property set to a number strictly greater than zero, Location: #/components/schemas/MyObject
```
which contradicts section 6.2.1 of the JSON schema validation specification which states 
> The value of "multipleOf" MUST be a number, strictly greater than 0.
> A numeric instance is valid only if division by this keyword's value results in an integer.

**Steps to Reproduce**

1. Create a schema with a multipleOf property greater than zero but less than one. This is eg. useful for money amounts: setting multipleOf to 0.01 prevents fractional cent values.
2. Start the server.

**Expected behavior**
As 0.01 is strictly greater than zero I expect no validation error in the log.

**Diagnostic information:**
 - OpenLiberty Version: 19.0.0.9
 - Java Version: openjdk version "13" 2019-09-17 OpenJDK Runtime Environment (build 13+33) OpenJDK 64-Bit Server VM (build 13+33, mixed mode, sharing)
 - server.xml configuration
```
<server desc="BUILDVAR::UNITDESC">
    <featureManager>
        <feature>beanValidation-2.0</feature>
        <feature>cdi-2.0</feature>
        <feature>concurrent-1.0</feature>
        <feature>ejbLite-3.2</feature>
        <feature>jaspic-1.1</feature>
        <feature>jaxrs-2.1</feature>
        <feature>jsonb-1.0</feature>
        <feature>jsonp-1.1</feature>
        <feature>localConnector-1.0</feature>
        <feature>mpConfig-1.3</feature>
        <feature>mpOpenAPI-1.0</feature>
        <feature>mpRestClient-1.2</feature>
        <feature>servlet-4.0</feature>
        <feature>jaxrsClient-2.1</feature>
        <feature>mpMetrics-1.1</feature>
        <feature>mpFaultTolerance-2.0</feature>
    </featureManager>

    <httpEndpoint host="*" httpPort="${env.EBKNG_DEFAULT_HTTP_PORT}" httpsPort="${env.EBKNG_DEFAULT_HTTPS_PORT}" id="defaultHttpEndpoint">
        <tcpOptions acceptThread="true" waitToAccept="true"/>
    </httpEndpoint>

    <applicationMonitor updateTrigger="mbean"/>

    <jndiEntry jndiName="auth.token.password" value="${env.EBKNG_AUTH_TOKEN_PASSWORD}"/>

    <webApplication contextRoot="BUILDVAR::WARCONTEXTROOT" id="BUILDVAR::UNITNAME" name="BUILDVAR::UNITNAME"  location="BUILDVAR::WARFILENAME"  />
</server>
```

**Additional context**
This seems to be traceable to com.ibm.ws.microprofile.openapi.impl.validation.SchemaValidator.java:60ff
```
            if (t.getMultipleOf() != null && (t.getMultipleOf().compareTo(BigDecimal.ONE) < 1)) {
                final String message = Tr.formatMessage(tc, "schemaMultipleOfLessThanOne");
                helper.addValidationEvent(new ValidationEvent(ValidationEvent.Severity.ERROR, context.getLocation(), message));
            }
```
where the value of the multipleOf property is compared to one instead of zero.

**Comment by Emily-Jiang** [comment]
@yasammez Thanks for raising this bug! Since you have done some investigation, would you like to do a PR? Of course, you need to sign CLA first, which should be quite straightforward.

**Comment by Azquelt** [comment]
This was fixed for all versions of the `mpOpenApi` feature by #18002 which was included in 21.0.0.9.
